src/main/java/edu/cornell/kfs/pmw/batch/PaymentWorksUtils.java

```diff
@@ -0,0 +1,40 @@
+package edu.cornell.kfs.pmw.batch;
+
+import org.apache.commons.lang3.StringUtils;
+import org.kuali.kfs.krad.util.ObjectUtils;
+import org.kuali.kfs.sys.KFSConstants;
+import org.kuali.kfs.vnd.VendorConstants;
+
+public final class PaymentWorksUtils {
+
```

Comment:
The following may be a possible solution to deal with both of our concerns:

a) Rename class PaymentWorksUtils to something that better represents what the class was created for; maybe  PaymentWorksDataTransformation 

b) Refactor the methods in the current PaymentWorksDataTransformationService and PaymentWorksDataTransformationServiceImpl to also be static methods in that newly created class from (a) above. Those existing methods in that service class do not call out to any other services so I believe that they can become static methods also. This refactoring could be performed as a new user story or could be done now. Full regression testing would be needed with this change since the methods in the current service class touch all three batch jobs. 

Your thoughts?

Reply:
Yes, I agree that your proposed refactoring makes sense and is feasible. Given that the PaymentWorksUtils class was introduced by this PR and the refactoring appears to be a minimal amount of work, I think it makes sense to include it in this PR. We already have a separate user story for full regression testing once the remaining critical Domestic DV Vendor tickets are resolved.